pkg/runtime.native.System.IO.Ports/runtime.native.System.IO.Ports.pkgproj

```diff
@@ -0,0 +1,15 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), Directory.Build.props))\Directory.Build.props" />
+  <PropertyGroup>
+    <SkipPackageFileCheck>true</SkipPackageFileCheck>
+    <SkipValidatePackage>true</SkipValidatePackage>
+    <SystemIOPortsNativePath>$(ArtifactsBinDir)\native\$(TargetGroup)-$(DefaultOSGroup)-$(Configuration)-$(ArchGroup)</SystemIOPortsNativePath>
+  </PropertyGroup>
+  <ItemGroup Condition="'$(OS)' != 'Windows_NT' and '$(PackageTargetRuntime)' != ''">
```

Comment:
Windows condition here is bad.   You are getting a Windows build because of this behavior: 
https://github.com/dotnet/corefx/blob/3685052201c95462f210582be9fe380baf1b902e/pkg/Directory.Build.props#L16-L24

The purpose is to let folks build a package on a platform that may not be officially suppoted yet. That may not be something you want. You can have a property that turns this itemgroup off and instead just sets buildrid to officialbuildrid.

Reply:
I think I'll go with explicit PackageTargetRuntime check

Reply:
Another thought: change the item name in your Rid.props file to use BuildRID instead of OfficialBuildRID and it will avoid this behavior.